app/views/layouts/ats/_navbar.html.slim

```diff
@@ -42,12 +40,7 @@ nav.navbar.navbar-expand-lg.align-items-center.fixed-top.header.gap-2
       .mt-1.d-lg-none
         ul.navbar-nav.gap-1
           li.dropdown-divider.my-2 role="separator"
```

Comment:
You're adding a custom style for that delimiter, although the problem is solved by simply using `hr`, instead of `li` (why `li` here anyway)